### ISSUE TITLE
theme upgrade breaks site

### ISSUE THREAD

**Issue by andrewfandrew** [Nikeo/customizr#67]
Hello Nikeo,
I have been using Customizr for several years. Sorry at this stage i do not have screenshots with error message. But i just tried to upgrade from old version of Custmizr to the latest. This was after upgrading to latest version of wordpress and also successfully upgrading the plugins. Although there are several themes, i left the others alone and merely upgraded Customizr. It breaks the site. 

Sorry i am not at the level where i can fix this issue. Sorry to sound negative, but at least i thought you would prefer to know.

Thanks,
Andrew

**Comment by Nikeo** [comment]
Thanks for reporting this Andrew  :)
You'll want to open a new topic on the support forum and give the following details 
- error messages?
- using a child theme ?
- version of WordPress (the latest as you said ?)
- did you try to desactivate all your plugin to see if there could be a collision ?

https://wordpress.org/support/theme/customizr

Thanks !

**Comment by andrewfandrew** [comment]
Thanks Nikeo, If i can test out will raise new topic as you ask me to. I can clone the site on my hosting provider (A2 Hosting). But when i try to upgrade the clone, i get redirected to the production site. I do not want to create more downtime on the production site so need to work out a way to test this issue out fully but without relying on the convenient site cloning available. Hope to put that together in a day or two for you. Will try the troubleshooting techniques you suggest and can post it in the forum. Just to say i used the built in wordpress upgrade function and started with a 4.0.1 to 4.1 Wordpress version upgrade, then the plugins, all went fine, then just attempted a (parent) customizr theme upgrade. I have created a child theme. I did get an error message like 'TC::'. 

I do use a child theme
